plugin.go

```diff
@@ -2,31 +2,32 @@
 package plasmactlbump
 
 import (
-	"github.com/spf13/cobra"
-
 	"github.com/launchrctl/keyring"
```

Comment:
- [/] Throw an error during propagation when domain repo contain unversioned changes in a group_var variable
      sed -i 's/account_port: .*/account_port: 8099/g' platform/group_vars/platform/vars.yaml
      compose and sync
```
ERROR   HEAD variable account_port from platform/group_vars/platform/vars.yaml doesn't much real version
> 
ERROR   variable account_port from platform/group_vars/platform/vars.yaml is an unversioned change. You need to commit this variable change.
```

Reply:
ERROR   variable account_port from platform/group_vars/platform/vars.yaml is an unversioned change. You need to commit this variable change.